src/main/java/com/moodybluez/enterprise/service/IMoodService.java

```diff
@@ -5,6 +5,7 @@
 public interface IMoodService {
     /**
      * fetch a mood with a given ID
+     *
```

Comment:
This is an extremely nit-picky Javadoc kind of thing. According to some of the best practice documentation I saw, and from what I've seen in other projects I've worked with; generally, people put a new line in between the description and the first @param. Again, like I said, this is very nit picky, just because the code is that good! [SOURCE](https://blog.joda.org/2012/11/javadoc-coding-standards.html)